Documentation/ApiOverview/SymfonyExpressionLanguage/Index.rst

```diff
@@ -90,8 +85,12 @@ the formerly registered PHP class name:
 Additional variables
 ====================
 
-Additional variables can already be provided within the
-:php:`CustomTypoScriptConditionProvider` PHP class:
+Additional variables can be provided by the registered provider class.
+In practice, adding additional variables are used rather seldom: To
```

Comment:
```suggestion
In practice, adding additional variables is used rather seldom: To
```

Reply:
several variables, therefore "are" is correct

Reply:
it's not about the variables, but about the "adding"

Reply:
Personally, I would go with David's change. But maybe @tomwarwick or @sarahmccarthy123 can shed light on us which one is the correct usage.

Reply:
In this context I would say "is" because it's "the adding" that is singular. You could say "additional variables are added rather seldom" then the "are" relates to "additional variables".

Reply:
Thank you for the clarification :-)